src/client/src/orangehrmMaintenancePlugin/components/SelectedCandidates.vue

```diff
@@ -0,0 +1,182 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-paper-container">
+    <div class="orangehrm-header-container">
+      <oxd-text tag="h6" :class="classes">
+        {{ $t('maintenance.selected_candidates') }}
+      </oxd-text>
+      <oxd-button
+        v-show="total > 0"
+        :label="$t('maintenance.purge_all')"
+        display-type="secondary"
+        @click="$emit('purge')"
+      />
+    </div>
+    <table-header :total="total" :loading="isLoading || loading"></table-header>
+    <div class="orangehrm-container">
+      <oxd-card-table
+        :headers="headers"
+        :clickable="false"
+        :selectable="false"
+        :loading="isLoading || loading"
```

Comment:
is this loading prop necessary here?

Reply:
yes because after pressing purge button I want to show loading spinner.